.github/linters/.markdownlint.yml

```diff
@@ -0,0 +1,51 @@
+---
+###########################
+###########################
+## Markdown Linter rules ##
```

Comment:
thanks for adding these 🎉 
@calebk1 do you know if there are many changes we need to make to conform with the new linter rules?

Reply:
Currently, the linter has issues with the links we do for tabs - but I'm discussing with @dmitrymatio to confirm what needs to happen there. Once that's all settled, I'll do a more thorough look at the situation.

Reply:
Is it the linter or link checker?

Reply:
Oh sorry! It's part of the validation: https://github.com/AdobeDocs/aep-mobile-sdkdocs/actions/runs/4693931374/jobs/8321523397?pr=300

Reply:
oh yeah the test pull request workflow runs the link checker. you can also run it locally with `test:links`

Reply:
It's not a blocking workflow but if it will trigger every time so maybe we disable it. We will have to discuss with team on monday.

Reply:
That is where my original question came from, if with the rules and current setup we see a lot of warnings do we plan to resolve them first and then enable both the linter and the link checker as a blocking workflows? Otherwise I bet we will forget to run these checks manually and fix the warnings as new changes go in.

Reply:
If we can get this tabs issue with links resolved, I would definitely like to turn on the link checker as a blocking workflow (since it's always easy to screw up links, so making sure those are correct is good).

As for the linter itself, I'll need to double check to see what's going on there to confirm if we want to turn that on or not (since sometimes the linter can catch odd things that aren't necessarily big issues).

Reply:
That sounds good @calebk1, I am curios to see how many warnings we see with these rules. Also I assume we can tweak the rules for our use-case in case we prefer or use another format consistently?